File: PRÁCTICA SEGUNDO PARCIAL/Ejercicio4/Java/SustituirEspacios.java
import java.util.Scanner;

public class SustituirEspacios {
    public static String sustituirEspacios(String cadena) {
        String nuevaCadena = cadena.replace(" ", "$$$$$");
        return nuevaCadena;
    }

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Ingrese una cadena: ");
        String cadena = scanner.nextLine();
        String cadenaSustituida = sustituirEspacios(cadena);
        System.out.println("Cadena original: " + cadena);
        System.out.println("Cadena sustituida: " + cadenaSustituida);
        scanner.close();
    }
}

File: PRÁCTICA SEGUNDO PARCIAL/Ejercicio4/Python/SustituirEspacios.py
def sustituir_espacios(cadena):
    nueva_cadena = cadena.replace(" ", "$$$$$")
    return nueva_cadena

cadena = input("Ingrese una cadena: ")
cadena_sustituida = sustituir_espacios(cadena)
print("Cadena original:", cadena)
print("Cadena sustituida:", cadena_sustituida)
